File: .sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { Bng2latlongConverterSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await Bng2latlongConverterSDK.test()
    equal(null !== testsdk, true)
  })

})

File: lua/test/exists_test.lua
-- Bng2latlongConverter SDK exists test

local sdk = require("bng2latlong-converter_sdk")

describe("Bng2latlongConverterSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
